app/editor/src/store/hooks/content/useContent.ts

```diff
@@ -0,0 +1,46 @@
+import { IContentFilter, IContentModel, IPaged } from 'hooks';
+import { useApiContents } from 'hooks/api-editor/editor';
+import React from 'react';
+import { IContentStore, useContentStore } from 'store/slices';
+import { IContentState } from 'store/slices/content';
+
+interface IContentHook {
+  getContent: (id: number) => Promise<IContentModel>;
+  findContent: (filter: IContentFilter) => Promise<IPaged<IContentModel>>;
+  addContent: (content: IContentModel) => Promise<IContentModel>;
+  updateContent: (content: IContentModel) => Promise<IContentModel>;
+  deleteContent: (content: IContentModel) => Promise<IContentModel>;
+}
+
+export const useContent = (): [IContentState, IContentHook, IContentStore] => {
```

Comment:
I may remove this hook since we're not storing anything in redux with it.  But if we were storing the results in redux we would have this separate hook to do it.